featurebyte/routes/target/controller.py

```diff
@@ -124,6 +130,24 @@ async def get_info(
         entity_brief_info_list = await self.entity_service.get_entity_brief_info_list(
             set(entity_ids)
         )
+
+        # Get input table metadata
+        assert (
+            len(target_doc.table_ids) == 1
+        ), "Target should have only one table for now, until forward joins are supported."
+        table_doc = await self.table_service.get_document(document_id=target_doc.table_ids[0])
+        input_data = InputData(
+            main_data=TableMetadata(
+                name=table_doc.name,
+                data_type=str(table_doc.type),
+            ),
+        )
+
+        # Get metadata
+        operation_structure = target_doc.graph.extract_operation_structure(target_doc.node)
```

Comment:
Is `target_doc` here an instance of `BaseFeatureModel`? If so it has a `extract_operation_structure` method that returns a `GroupOperationStructure` like feature. Can that be reused?